languages/python/sqlalchemy-oso/sqlalchemy_oso/auth.py

```diff
@@ -5,23 +5,25 @@
 from sqlalchemy.orm.session import Session
 from sqlalchemy.sql import expression as sql
 
-from sqlalchemy_oso.partial import partial_to_query
+from sqlalchemy_oso.partial import partial_to_query, partial_to_filter
 
-def null_query(session: Session) -> Query:
+
+def null_query(session: Session, model) -> Query:
     """Return an intentionally empty query."""
     # TODO (dhatch): Make this not hit the database.
```

Comment:
Do you have a plan for this?

Reply:
To make it not hit the database? Not precisely. Could have a special sentinel object we return that indicates to the calling code to return no results.  But ultimately it would require a special SQLAlchemy query subclass that doesn't hit the db. Might be tough.